provider/data/collator/data/collation_reordering_v1_marker.rs.data

```diff
@@ -42,7 +42,8 @@ macro_rules! __impl_collation_reordering_v1_marker {
+                use icu_provider_baked::binary_search::tinystr::tinystr;
+                icu_provider_baked::binary_search::Data(&[(tinystr!(19usize, "am"), __AM), (tinystr!(19usize, "ar"), __AR), (tinystr!(19usize, "as"), __AS), (tinystr!(19usize, "az"), __AZ), (tinystr!(19usize, "be"), __BE), (tinystr!(19usize, "bg"), __BE), (tinystr!(19usize, "bn"), __AS), (tinystr!(19usize, "bs"), __AZ), (tinystr!(19usize, "bs-Cyrl"), __BE), (tinystr!(19usize, "chr"), __CHR), (tinystr!(19usize, "el"), __EL), (tinystr!(19usize, "fa"), __AR), (tinystr!(19usize, "gu"), __GU), (tinystr!(19usize, "he"), __HE), (tinystr!(19usize, "hi"), __HI), (tinystr!(19usize, "hr"), __AZ), (tinystr!(19usize, "hy"), __HY), (tinystr!(19usize, "ja"), __JA), (tinystr!(19usize, "ka"), __KA), (tinystr!(19usize, "kk"), __BE), (tinystr!(19usize, "km"), __KM), (tinystr!(19usize, "kn"), __KN), (tinystr!(19usize, "ko"), __KO), (tinystr!(19usize, "kok"), __HI), (tinystr!(19usize, "ku"), __KU), (tinystr!(19usize, "ky"), __BE), (tinystr!(19usize, "lo"), __LO), (tinystr!(19usize, "mk"), __BE), (tinystr!(19usize, "ml"), __ML), (tinystr!(19usize, "mn"), __MN), (tinystr!(19usize, "mr"), __HI), (tinystr!(19usize, "my"), __MY), (tinystr!(19usize, "ne"), __NE), (tinystr!(19usize, "or"), __OR), (tinystr!(19usize, "pa"), __PA), (tinystr!(19usize, "ps"), __AR), (tinystr!(19usize, "ru"), __BE), (tinystr!(19usize, "si"), __SI), (tinystr!(19usize, "sr"), __BE), (tinystr!(19usize, "sr-Latn"), __AZ), (tinystr!(19usize, "ta"), __TA), (tinystr!(19usize, "te"), __TE), (tinystr!(19usize, "th"), __TH), (tinystr!(19usize, "ug"), __AR), (tinystr!(19usize, "uk"), __BE), (tinystr!(19usize, "ur"), __AR), (tinystr!(19usize, "yue-u-co-stroke"), __YUE_U_CO_STROKE), (tinystr!(19usize, "yue-u-co-unihan"), __YUE_U_CO_STROKE), (tinystr!(19usize, "yue-u-co-zhuyin"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh"), __ZH), (tinystr!(19usize, "zh-Hant-u-co-stroke"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh-Hant-u-co-unihan"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh-Hant-u-co-zhuyin"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh-u-co-stroke"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh-u-co-unihan"), __YUE_U_CO_STROKE), (tinystr!(19usize, "zh-u-co-zhuyin"), __YUE_U_CO_STROKE)])
```

Comment:
Thought: This will improve when we move `-u-co-xxx` to DataMarkerAttributes... we haven't done that yet?

Reply:
No, this was part of https://github.com/unicode-org/icu4x/pull/4984

Reply:
Ok but we are still planning to do it?